test/e2e/tls/tls.go

```diff
@@ -79,7 +80,7 @@ func checkTLS(f *framework.Framework, server string) {
 	createdPod := podClient.Create(pod)
 	defer func() {
 		By("delete the pod")
-		podClient.DeleteSync(createdPod.Name, &metav1.DeleteOptions{}, framework.DefaultPodDeletionTimeout)
+		podClient.DeleteSync(createdPod.Name, metav1.DeleteOptions{}, framework.DefaultPodDeletionTimeout)
```

Comment:
what is reason here, is it fix to problem, or something else? change is other way as similar change above.

Reply:
An API change in client-go.